src/lib/messages.js

```diff
@@ -0,0 +1,91 @@
+import { storage } from 'sdk/simple-storage';
+import tabs from 'sdk/tabs';
+import { getMostRecentBrowserWindow } from 'sdk/window/utils';
+
+import measure from '../measurements';
+import { e10sProcessCount, e10sStatus } from './e10s-status';
+import Logger from './log';
+import sendEvent from './metrics';
+
+const logger = new Logger(
+  'sdk.lib.messages',
+  getMostRecentBrowserWindow().console
+);
+
+export const storeTab = uuid => {
```

Comment:
It keeps scope clearer if you default to arrow functions unless there's an explicit reason to use function form.